README.rst

```diff
@@ -4,24 +4,35 @@
 .. image:: https://img.shields.io/pypi/pyversions/twine.svg
     :target: https://pypi.org/project/twine
 
-.. image:: https://img.shields.io/readthedocs/twine
-    :target: https://twine.readthedocs.io
-
 .. image:: https://img.shields.io/github/workflow/status/pypa/twine/Main
     :target: https://github.com/pypa/twine/actions
 
 .. image:: https://img.shields.io/codecov/c/github/pypa/twine
     :target: https://codecov.io/gh/pypa/twine
 
+.. docs-badges-end
```

Comment:
Maybe have something like this?
```suggestion
.. docs-badges-end-for-docs-inclusion
```

I'm pretty sure many folks don't know about the partial inclusion feature and that's why I feel like it's important to put out some sort of a warning in places with markers.